Add `4 Digit osu!mania World Cup` wiki article (#7978)
ppy/osu-wiki: wiki/Tournaments/4DM/1/en.md

---
tags:
  - 4DM
  - 4DMWC
---

# 4 Digit osu!mania World Cup

> The **4 Digit osu!mania World Cup** (***4DM***) was a worldwide country-based double-elimination 3v3 osu!mania 4-key tournament hosted by ::{ flag=GB }:: [Fletch](https://osu.ppy.sh/users/8946085). Players will form a team with all its members ranked between #1,000 and #9,999 were able to participate. It was the first instalment of the 4 Digit osu!mania World Cup.

Reviewer: ```suggestion
The **4 Digit osu!mania World Cup** (***4DM***) was a worldwide country-based double-elimination 3v3 osu!mania 4-key tournament hosted by ::{ flag=GB }:: [Fletch](https://osu.ppy.sh/users/8946085). Players formed teams with all members being ranked between #1,000 and #9,999. It was the first instalment of the 4 Digit osu!mania World Cup.
```

- LN
  1. [Toby Fox - Legend (RuleBlazing) \[3 Heroes \[edit\]\]](https://osu.ppy.sh/beatmapsets/875919#mania/2059082)
  2. [ginkiha - nightfall (Abraxos) \[rave's dream\]](https://osu.ppy.sh/beatmapsets/473053#mania/1010821)
- SV
  1. [X&G - Control (tailsdk) \[Stage 1: Control\]](https://osu.ppy.sh/beatmapsets/791643#mania/1660728)
  2. [Lily Allen - Fuck You (DJ THT Radio Edit) (XeoStyle) \[4k MX (Hard SV Edit.)\]](https://osu.ppy.sh/beatmapsets/314482#mania/701663)
- Tiebreaker
  1. **[Magnetude - Broken (Salty Mermaid) \[Shattered\]](https://osu.ppy.sh/beatmapsets/842099#mania/1762138)**

### Qualifiers

- FreeMod
  1. [THE SxPLAY & KIVA - Hyou Ryuu (zero2snow) \[Stage 1: Drifted\]](https://osu.ppy.sh/beatmapsets/980182#mania/2051288)
  2. [Nekomata Master - Sayonara Heaven (\[Crz\]Crysarlene) \[Stage 2 : Tranquility\]](https://osu.ppy.sh/beatmapsets/980130#mania/2051174)
  3. [OISHII - CELLS AT WORK (HowToPlayLN) \[Stage 3 : Metabolism\]](https://osu.ppy.sh/beatmapsets/980039#mania/2050996)
  4. [Thaehan - Final Boss (Theresa May) \[Stage 4 : Credits\]](https://osu.ppy.sh/beatmapsets/980090#mania/2051097)

## Match results

> *All match links were unavailable to be obtained, due to lack of documentation.*

Reviewer: ```suggestion
*All match links are unavailable, due to lack of documentation.*
```

### Grand Finals

Saturday, 27 July 2019:

| Team 1 |  |  | Team 2 |
| --: | :-: | :-: | :-- |
| Philippines ::{ flag=PH }:: | 3 | **7** | ::{ flag=AR }:: **Argentina** |
| **United States** ::{ flag=US }:: | **7** | 6 | ::{ flag=AR }:: Argentina |

### Finals

Saturday, 20 July 2019:

| Team 1 |  |  | Team 2 |
| --: | :-: | :-: | :-- |
| **China** ::{ flag=CN }:: | **7** | 5 | ::{ flag=FR }:: France |

> | Singapore ::{ flag=SG }:: | 1 | **7** | ::{ flag=AR }:: Argentina |

Reviewer: ```suggestion
| Singapore ::{ flag=SG }:: | 1 | **7** | ::{ flag=AR }:: **Argentina** |
```
